### PR TITLE
Move CephFS test container to test/images.

### PR DESCRIPTION
**Pull request by jsafrane**
From some reason the container was still in contrib/for-tests/volumes-tester.

### CODE REVIEW

File: test/images/volumes-tester/ceph/README.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ Used by test/e2e/* to test CephFSVolumeSource. Not for production use!
 
 
 [![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/contrib/for-tests/volumes-tester/ceph/README.md?pixel)]()
```
Comment (ixdy) [inline test/images/volumes-tester/ceph/README.md]: Looks like this needs to be fixed, possibly manually?

**Comment by ixdy** [comment]
@k8s-bot unit test this please

File: test/images/volumes-tester/ceph/README.md (hunk 1 above)
Comment (jsafrane) [inline test/images/volumes-tester/ceph/README.md]: Good catch, fixed.

**Comment by ixdy** [comment]
LGTM

@k8s-bot unit test this please

**Comment by jsafrane** [comment]
Can anyone please upload this new image to gcr.io/google_containers/? I do not have privileges to do so. @saad-ali?

```
cd test/images/volumes-tester/ceph/
make push
```

**Comment by saad-ali** [comment]
@jsafrane `make push` fails with:
```BASH
GPG key retrieval failed: [Errno 12] Timeout on https://git.ceph.com/?p=ceph.git;a=blob_plain;f=keys/release.asc: (28, 'Operation too slow. Less than 1 bytes/sec transfered the last 30 seconds')
Error: Package: 1:ceph-0.94.5-0.el6.x86_64 (Ceph)
           Requires: liblttng-ust.so.0()(64bit)
Error: Package: gperftools-libs-2.0-11.el6.3.x86_64 (Ceph)
           Requires: libunwind.so.8()(64bit)
Error: Package: 1:librbd1-0.94.5-0.el6.x86_64 (Ceph)
           Requires: liblttng-ust.so.0()(64bit)
Error: Package: 1:librados2-0.94.5-0.el6.x86_64 (Ceph)
           Requires: liblttng-ust.so.0()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
The command '/bin/sh -c /usr/local/bin/install.sh' returned a non-zero code: 1
make: *** [container] Error 1
```

**Comment by jsafrane** [comment]
Some network hiccup? It works fine today.

**Comment by saad-ali** [comment]
Yep, looks like it went through this time: `gcr.io/google_containers/volume-ceph:0.1`

**Pull request merged**